src/Apps/Settings/Routes/EditProfile/Components/SettingsEditProfileFields.tsx

```diff
@@ -46,162 +51,186 @@ const SettingsEditProfileFields: React.FC<SettingsEditProfileFieldsProps> = ({
 }) => {
   const { sendToast } = useToasts()
   const { submitUpdateMyUserProfile } = useUpdateMyUserProfile()
+  const { relayEnvironment } = useSystemContext()
 
-  const {
-    values,
-    isSubmitting,
-    handleSubmit,
-    setFieldValue,
-    handleChange,
-  } = useFormik<EditProfileFormModel>({
```

Comment:
Refactored from a hook to a component to enable `useFormikContext` in a child component